cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractAccountApiTest.java

```diff
@@ -308,9 +345,20 @@ private ApiResponse createUser(List<String> subAccountsIds) throws Exception {
         return createUser(subAccountsIds, Account.Role.BILLING);
     }
 
+    private ApiResponse createUser(Boolean enabled, List<String> subAccountsIds) throws Exception {
+        return createUser(enabled, subAccountsIds, Account.Role.BILLING);
+    }
+
     private ApiResponse createUser(List<String> subAccountsIds, Account.Role role) throws Exception {
         String email = String.format("%s@%s.com", randomLetters(), randomLetters());
-        ApiResponse user = account.createUser("TestUserJava"+new Date().toString(), email, role, subAccountsIds, null);
+        ApiResponse user = account.createUser("TestUserJava"+new Date().toString(), email, role, null, subAccountsIds, null);
+        createdUserIds.add(user.get("id").toString());
+        return user;
+    }
+
+    private ApiResponse createUser(Boolean enabled,List<String> subAccountsIds, Account.Role role) throws Exception {
```

Comment:
Missing space here as well. Maybe try to change settings of your editor?

Reply:
Fixed, will look for IDE improvement